testapi.pm

```diff
@@ -475,22 +475,23 @@ subroutine block.
     send_key 'esc';
   }, 15);
 
-Returns true if screen changed or C<undef> on timeout. Default timeout is 10s.
+Returns true if screen changed or false on timeout. Default timeout is 10s. Default
+similarity_level is 50.
 
 =cut
 
 sub wait_screen_change(&@) {
-    my ($callback, $timeout) = @_;
-    $timeout ||= 10;
+    my $callback         = shift;
+    my $timeout          = shift || 10;
+    my $similarity_level = shift || 50;
```

Comment:
I would prefer unpacking the array list and using a hash for the more optional arguments, e.g.

```perl
my ($callback, $timeout, %args) = @_;
$timeout ||= 10;
$args{similarity_level} //= 50;
```